modules/nvidia_plugin/tests/functional/shared_tests_instances/set_device_name.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "set_device_name.hpp"
```

Comment:
Please, apply the formatting for `nvidia_plugin`:
```bash
cd ./nvidia_plugin
./utils/format.sh
```

Reply:
thanks a lot 👍